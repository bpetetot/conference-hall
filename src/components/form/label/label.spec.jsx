/* eslint-env jest */
import React from 'react'
import Label from './index'

const props = {
  label: 'label',
  name: 'name',
  error: 'error',
  className: 'blue',
  classNameInput: 'red',
}

describe('components/label', () => {
  it('should render', () => {
    expect(<Label {...props}>children</Label>).toMatchSnapshot()
  })
})
