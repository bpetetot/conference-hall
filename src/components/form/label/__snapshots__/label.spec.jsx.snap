// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/label should render 1`] = `
<Label
  className="blue"
  classNameInput="red"
  error="error"
  label="label"
  name="name"
>
  children
</Label>
`;
